lambdas/src/compile.rs

```diff
@@ -0,0 +1,177 @@
+use aws_config::BehaviorVersion;
+use aws_sdk_dynamodb::{error::SdkError, operation::put_item::PutItemError};
+use lambda_http::{
+    run, service_fn, Error as LambdaError, Request as LambdaRequest, Response as LambdaResponse,
+};
+use serde::Deserialize;
+use std::ops::Add;
+use tracing::{error, info};
+
+mod common;
+use crate::common::{errors::Error, utils::extract_request, Item, Status, BUCKET_NAME_DEFAULT};
+
+const QUEUE_URL_DEFAULT: &str = "https://sqs.ap-southeast-2.amazonaws.com/266735844848/zksync-sqs";
```

Comment:
I think we should set it via `env`

Reply:
It's done [here](https://github.com/NethermindEth/zksync-remix-plugin/blob/a9d07097555df71776143b91811f95caf1211bef/lambdas/src/compile.rs#L149). that is just default for ease of use for now. Will add comment to remove

Reply:
done